### ISSUE TITLE
When sending multiple pictures at the same time, the prediction result of the same picture will be different every time

### ISSUE THREAD

**Issue by xiaoxiaoshu111** [triton-inference-server/server#3090]
**Description**
I export model to torchscript and try to deploy in triton server
my config.pbtxt is below
![image](https://user-images.githubusercontent.com/29574038/124756643-448e5d80-df5f-11eb-9ab3-83f8b882f4af.png)

**Triton Information**
tritonserver_r20.11

**Expected behavior**
When only one picture is predicted at a time, the result is correct. But when predicting multiple pictures at the same time, the results returned are different each time.

I get the torchscript model by [https://github.com/triton-inference-server/server/commit/2ff921a6b483a73fbaa9f3a14b4edf060352d70c](url)

How should I solve this problem? 
thank you for reading

**Comment by dzier** [comment]
Can you include the detailed output log from Triton along with the reproduction steps?

**Comment by CoderHam** [comment]
@xiaoxiaoshu111 which version of PyTorch did you use to produce the model. It looks like you did not set the eval mode before tracing the model. 
```
model.eval()
```
> But when predicting multiple pictures at the same time, the results returned are different each time.
There may be an issue in your client code. Can you share the client you are running in your experiments or a minimal repro for the issue?

**Comment by xiaoxiaoshu111** [comment]
@CoderHam Wow，when I set the eval mode before tracing the model，it worked. Thank you very much for your suggestion！！！

And now I have a new problem，when I  use resnet50 torchscript model to deploy in triton server，the result is right. But when I use swinstranformer torchscript model to deploy in triton server，the prediction result of the same picture will be different every time. Is it a problem with some layers of the swinstranformer model? 

When I produce the swinstranformer model, I got these warnings：
`/usr/lib64/python3.6/dist-packages/torch/tensor.py:593: RuntimeWarning: Iterating over a tensor might cause the trace to be incorrect. Passing a tensor of different shape won't change the number of iterations executed (and might lead to errors or silently give incorrect results).
  'incorrect results).', category=RuntimeWarning)
/data/app/model_child_trt_server_multi/swintransform/swin_transformer.py:495: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert H == self.img_size[0] and W == self.img_size[1], \
/data/app/model_child_trt_server_multi/swintransform/swin_transformer.py:92: TracerWarning: Converting a tensor to a Python integer might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  print(windows.shape,"windows.shape")
/data/app/model_child_trt_server_multi/swintransform/swin_transformer.py:93: TracerWarning: Converting a tensor to a Python integer might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  B = int(windows.shape[0] / (H * W / window_size / window_size))
/data/app/model_child_trt_server_multi/swintransform/swin_transformer.py:355: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert L == H * W, "input feature has wrong size"`

Any suggestion will be appreciated.

Thanks again！
